dunwoody_disaster/Item.py

```diff
@@ -29,9 +29,10 @@ def __repr__(self) -> str:
 
 
 class Weapon(Item):
-    def __init__(self, name, damage, magicCost, staminaCost):
+    def __init__(self, name, magic, damage, magicCost, staminaCost):
         super().__init__(name)
         self.stats = WeaponStats[name]
+        self.magic = magic
         self.damage = damage
         self.magicReq = magicCost
```

Comment:
Why is there magic and magicCost?

Reply:
magic cost is the amount of mana the item uses

Reply:
magic is just a boolean that shows if an item does magic damage or not

Reply:
Aha. if only there was a type hint, I might've seen that!

Reply:
If you petition the switch to java I will make sure everything has type hints

Reply:
Lol why would I do that

Reply:
I'm petitioning we switch to Rust.